src/Lib/Twig/Loader.php

```diff
@@ -57,10 +75,29 @@ public function getCacheKey($name)
     public function isFresh($name, $time)
     {
         $name = $this->resolveFileName($name);
+
         return filemtime($name) < $time;
     }
 
     /**
+     * Check if we have the source code of a template, given its name.
+     *
+     * @param string $name The name of the template to check if we can load
+     *
+     * @return bool If the template source code is handled by this loader or not
+     */
+    // @codingStandardsIgnoreStart
+    public function exists($name) {
```

Comment:
Could you put the opening brace `{` on a new line?

Reply:
Will fix all those, too; i think we miss a cs checker?

Reply:
Well if we're going PHP 7, we might as well start using `api-clients/test-utilities` in `require-dev` and use everything getting pulled in by that. Including Friends of PHP CS fixer

Reply:
What is 'codingStandardsIgnoreStart' for - the brace?

Reply:
Can't remember in this case tbh

Reply:
I have added `"api-clients/resource-test-utilities": "^1.0.0",` but it required a ton-load of vendor dependencies

Reply:
Willl remove the condingStandardsIgnore passages

Reply:
> and use everything getting pulled in by that. Including Friends of PHP CS fixer

No idea how - though.

Reply:
I can fix that, will look into making a PR to your PR tomorrow. Also `^4.1` is the latest 😉 . Take a look at these files if you want to dive into it though:

- https://github.com/php-api-clients/travis/blob/master/composer.json#L62-L114
- https://github.com/php-api-clients/travis/blob/master/Makefile
- https://github.com/php-api-clients/travis/blob/master/.php_cs

Reply:
Which package 4.1? https://packagist.org/packages/api-clients/resource-test-utilities <= ^1.0.0?

Reply:
This one: https://packagist.org/packages/api-clients/test-utilities

Reply:
I have added those.
* I could fix parallel-lint.
* hangs here
```
Fatal error: Uncaught Error: Class 'ApiClients\Tools\TestUtilities\PhpCsFixerConfig' not found in /var/www/cakephp/app/vendor/wyrihaximus/twig-view/.php_cs on line 11
```

Reply:
Ah and yeah I had to downgrade phpunit to 6.2 because:
```

  Problem 1
    - friendsofphp/php-cs-fixer v2.5.0 requires sebastian/diff ^1.4 -> satisfiable by sebastian/diff[1.4.3, 1.4.0, 1.4.1, 1.4.2].
    - friendsofphp/php-cs-fixer v2.5.0 requires sebastian/diff ^1.4 -> satisfiable by sebastian/diff[1.4.3, 1.4.0, 1.4.1, 1.4.2].
    - Can only install one of: sebastian/diff[2.0.1, 1.4.0].
    - Can only install one of: sebastian/diff[2.0.1, 1.4.1].
    - Can only install one of: sebastian/diff[2.0.1, 1.4.2].
    - Can only install one of: sebastian/diff[2.0.1, 1.4.3].
    - Can only install one of: sebastian/diff[2.0.1, 1.4.3].
    - phpunit/phpunit 6.3.0 requires sebastian/diff ^2.0 -> satisfiable by sebastian/diff[2.0.1].
    - Installation request for phpunit/phpunit ^6.3.0 -> satisfiable by phpunit/phpunit[6.3.0].
    - Installation request for friendsofphp/php-cs-fixer ^2.5.0 -> satisfiable by friendsofphp/php-cs-fixer[v2.5.0].
```

Reply:
Yeah you need https://packagist.org/packages/api-clients/test-utilities for that first error to go away.

Reply:
And phake/phake from ^3.0.0 to ^2.3.0
```

  Problem 1
    - Installation request for api-clients/test-utilities ^4.1.0 -> satisfiable by api-clients/test-utilities[4.1.0].
    - api-clients/test-utilities 4.1.0 requires phake/phake ^2.3 -> satisfiable by phake/phake[v2.3.0, v2.3.1, v2.3.2] but these conflict with your requirements or minimum-stability.
```

I tried to tweak minimum stability (tried all values @ https://getcomposer.org/doc/04-schema.md#minimum-stability) but I cannot get a sufficient dependency ok from cromposer...

Reply:
I have removed `"api-clients/test-utilities": "^4.1.0",`  (and resource-test-utilities) again... can you take a look tomorrow, with adding test-utilities? Cause I get a lot of conflicts and you may be able to just fix the packages.

Reply:
Sure 👍